src/model/token/mod.rs

```diff
@@ -0,0 +1,22 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+pub mod revoke;
+pub mod test;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct TokenTestResults {
+    #[serde(flatten)]
+    pub inner: HashMap<String, TokenTestResult>,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TokenTestResult {
```

Comment:
I believe this might work and be more readable.

```suggestion
pub type TestResults = HashMap<String, Token>;

#[derive(Clone, Debug, Deserialize, Serialize)]
#[serde(rename_all = "camelCase")]
pub struct Token {
```

Reply:
Changed, will report when I test it.

Reply:
This seems to have worked

Reply:
Never mind, turns out I never tried this method yet.